src/js/core.js

```diff
@@ -130,6 +130,21 @@
             node.parentElement.removeChild(node);
 
             event.preventDefault();
+        } else if (MediumEditor.util.isKey(event, MediumEditor.util.keyCode.BACKSPACE) &&
+                (tagName === 'blockquote' || node.parentElement.nodeName.toLowerCase() === 'blockquote') &&
+                MediumEditor.selection.getCaretOffsets(node).left === 0) {
+
+            // when cursor is at the begining of the element and the element is <blockquote>
+            // then pressing backspace key should change the <blockquote> to a <p> tag
+            event.preventDefault();
+
+            //if <blockquote> is the parent element, then make it the current node
+            if (node.parentElement.nodeName.toLowerCase() === 'blockquote') {
```

Comment:
I think I may have steered you in the wrong direction here.  I believe calling `execFormatBlock()` will work without you needing to actually select the entire block.  So, you might be able to remove this entire conditional and just have these 2 lines:

```js
event.preventDefault();
MediumEditor.util.execFormatBlock(this.options.ownerDocument, 'p');
```

Could you double check to see if that works?  If so, that will make this code nice an concise.  Apologies again for giving some bad info :worried: